template/data/src/test/java/co/nimblehq/template/data/repository/RepositoryTest.kt

```diff
@@ -9,6 +9,7 @@ import io.kotest.matchers.shouldBe
 import io.mockk.coEvery
```

Comment:
Please remove `import io.kotest.assertions.throwables.shouldThrow`

Reply:
@Wadeewee Fixed in https://github.com/nimblehq/android-templates/pull/292/commits/0d2563c9390e5a2bdfc1ae122af7d966f7093594